sympy/solvers/tests/test_inequalities.py

```diff
@@ -486,3 +486,47 @@ def test__pt():
     assert _pt(x, oo) == _pt(oo, x) == x + 1
     assert _pt(x, -oo) == _pt(-oo, x) == x - 1
     raises(ValueError, lambda: _pt(Dummy('i', infinite=True), S.One))
+
+
+def test_linear_programming():
```

Comment:
In my GSoC meeting with Aaron, he said there should be better test coverage.
https://docs.sympy.org/dev/contributing/new-contributors-guide/writing-tests.html#test-coverage

Reply:
I haven't been able to get the coverage functions to run without stalling or crashing. 

I tried 
`./bin/coverage_[report.py](http://report.py/) sympy/solvers` (stalls without making progress for at least an hour)
and 
`bin/coverage_report.py sympy/solvers/tests/test_inequalities.py` (crashes)

Despite this, I have added a lot of tests.